README.md

```diff
@@ -397,12 +412,12 @@ $server = Server::createServer($app2 /* ... */);
 
 In the above, if the URI of the original incoming request is `/root/foo`, what `$fooCallback` will receive is a URI with a past consisting of only `/foo`. This practice ensures that middleware can be nested safely and resolve regardless of the nesting level.
 
-If you want access to the full URI -- for instance, to construct a fully qualified URI to your current middleware -- `Phly\Conduit\Http\Request` contains a method, `getOriginalRequest()`, which will always return the original request provided:
+If you want access to the full URI -- for instance, to construct a fully qualified URI to your current middleware -- `Phly\Conduit\Http\Request` contains a method, `getOriginalRequest()`, which will always return the original request provided to the application:
 
 ```php
 function ($request, $response, $next)
 {
-    $location = $request->getOriginalRequest()->getAbsoluteUri() . '/[:id]';
+    $location = $request->getOriginalRequest()->getUri()->getPath() . '/[:id]';
```

Comment:
I will. I'm not 100% sure we can solve all 3 use cases, though -- the point that normal form of a URI *always* includes a "/" prefix is a good one, and makes this quite difficult to manage (as stripping a path segment cannot then logically leave us with an empty path). I'll see what I can do, though.